app/logic/Mail/AutoConfig/readConfig.ts

```diff
@@ -69,13 +69,17 @@ function readServer(xml: any, displayName: string, fullXML: any, source: ConfigS
   if (protocol == "exchange") { // Backwards compat
     let ewsURL = sanitize.url(xml.ewsURL, null);
     let owaURL = sanitize.url(xml.owaURL, null);
+    if (owaURL.startsWith("https://outlook.office365.com/")) {
+      owaURL = owaURL.replace("365", "");
+    }
     let activeSyncURL = sanitize.url(xml.easURL, null);
     assert(ewsURL || owaURL || activeSyncURL, "Malformed or unknown Exchange config");
     assert(ewsURL || owaURL, "ActiveSync not yet supported");
     // Return only the most preferred config (for this backwards compat)
-    protocol = ewsURL ? "ews" : owaURL ? "owa" : "activesync";
-    xml.url = ewsURL ?? owaURL ?? activeSyncURL;
+    protocol = owaURL ? "owa" : ewsURL ? "ews" : "activesync";
+    xml.url = owaURL ?? ewsURL ?? activeSyncURL;
```

Comment:
I'd like to default to EWS for Mustang, if both are available, given that EWS is faster with sync and therefore needs less resources.

Reply:
I didn't mean to commit this either, this was just so I could actually set up an account for test purposes.

Reply:
OK. Good. I wasn't sure whether it was for testing or intentional.

The "remove '365'" change is needed, though, right?

Reply:
I don't know how you want to handle the difference between the Office 365 OWA and EWS hostnames. If you're not going to default to OWA for Office 365, it doesn't really matter.